stats/gRPC.md

```diff
@@ -66,6 +69,11 @@ The following set of views are considered minimum required to monitor client-sid
 | grpc.io/client/roundtrip_latency      | grpc.io/client/roundtrip_latency      | distribution | grpc_client_method                     |
 | grpc.io/client/completed_rpcs         | grpc.io/client/roundtrip_latency      | count        | grpc_client_method, grpc_client_status |
 | grpc.io/client/started_rpcs           | grpc.io/client/started_rpcs           | count        | grpc_client_method                     |
+| grpc.io/client/sent_message_bytes     | grpc.io/client/sent_message_bytes     | distribution | grpc_client_method                     |
```

Comment:
You'd probably want the names to be consistent with the existing metrics such as "sent_bytes_per_rpc". So this would be "sent_bytes" instead.

Reply:
I wanted it to be clear that these are recorded *for each message* rather than for the whole RPC.

Reply:
"sent_message_bytes" doesn't make that more clear. To a random reader, the *message* here may be (mis)taken as describing the source of the bytes (message rather than metadata) instead of the granularity at which they are counted.

With the same idea as "sent_bytes_per_rpc", I would suggest "sent_bytes_per_message"

Reply:
FYI the names are updated.